src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApiAccessController.php

```diff
@@ -38,6 +39,9 @@
  */
 class ApiAccessController extends FrameworkBundleAdminController
 {
+    /**
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))")
```

Comment:
This route and other must be checked by @mflasquin who worked on these new Symfony pages 😉 .

@mflasquin as you already reviewed I guess it means that's OK for you?